tests/rig/tests/test_groups.py

```
@@ -383,3 +383,85 @@ def test_extra_manifest_missing_path_names_env_var(
+        defaults:
+          postgres_migrate:
+            workdir: workers
+        groups:
+          a:
+            tier: integration
+            paths: [x]
+            optional: true
+            requires_services: [postgres]
+        """,
+    )
+    with pytest.raises(ValueError, match="manage.py"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)